src/tool_help.c

```diff
@@ -80,7 +80,7 @@ static const struct helptxt helptext[] = {
    "Connect to host"},
   {"-C, --continue-at <offset>",
    "Resumed transfer offset"},
-  {"-b, --cookie <data>",
+  {"-b, --cookie <data|file>",
```

Comment:
Wouldn't "filename" be better for consistency with "-c"?

Reply:
It would indeed. Thanks, I'll amend...